staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -222,6 +222,7 @@ func (p *Patcher) buildMergePatch(original, modified, current []byte) ([]byte, e
 // gvkSupportsPatchOpenAPIV3 checks if a particular GVK supports the patch operation.
 // It returns an error if the OpenAPI V3 could not be downloaded.
 func (p *Patcher) gvkSupportsPatchOpenAPIV3(gvk schema.GroupVersionKind) (bool, error) {
+	// Bypassing root to save apiserver memory.
```

Comment:
That doesn't by-pass the root though. And I think that's fine. We usually apply a lot more than 1 resource types, going through the root might avoid a LOT with the cache busting.